sap/stackstate_checks/sap/sap.py

```diff
@@ -57,6 +59,8 @@ def _get_config(self, instance):
         self.verify = instance.get("verify", True)
         self.cert = instance.get("cert", "")
         self.keyfile = instance.get("keyfile", "")
+        self.domain = instance.get("domain", "")
+        self.stackstate_environment = instance.get("environment", "")
```

Comment:
@vishalxebia If we don't set the instance domain and environment it will default to empty strings instead of None. Can we keep it None instead such that we can decide whether we want to send it to StackState or not and have the StackPack put the default value. Or put the default value in here instead of an empty string

Reply:
@zandrewitte I had put `None` earlier but not sure what it is translated into JSON response for the replay. So I reverted and put back to an empty string. For an empty string, I am sure it works properly.